tests/integration/utils_test.go

```diff
@@ -289,11 +297,29 @@ func getAgentActionCmdOut(action *porterv1.AgentAction, aaOut string) string {
 that does "porter credentials list" will return just the result of the porter command from the job logs. This can be
 used to run porter commands inside the cluster to validate porter state
 */
-func runAgentAction(ctx context.Context, actionName string, namespace string, cmd []string) string {
-	aa := newAgentAction(namespace, actionName, cmd)
+func runAgentActionWithDefaultAgentCfg(ctx context.Context, actionName string, namespace string, cmd []string) string {
+	return runAgentAction(ctx, actionName, namespace, "", cmd)
+}
+
+func runAgentAction(ctx context.Context, actionName string, namespace string, agentCfgName string, cmd []string) string {
```

Comment:
I am not seeing where we use this function other than runAgentActionWithDefaultConfig. Did you have plans to support specifying different config and then ended up not needing it?

Reply:
Oh, I'm using it here: https://github.com/getporter/operator/pull/153/files#diff-50115afe52184bcb5d2d1627f4782a72c843b05c99174a6b2a4efe0525a2be66R34

Reply:
Oops, sorry I missed that. 👍